estate/models/estate_property_tag.py

```diff
@@ -0,0 +1,7 @@
+from odoo import fields,models
+
+class EsatePropertyTag(models.Model):
+    _name='estate.property.tag'
+    _description='Estate Property Tag'
+
+    name=fields.Char()
```

Comment:
```suggestion
    _name = 'estate.property.tag'
    _description = 'Estate Property Tag'

    name = fields.Char()
```
check the definition of field declaration in the codebase
for every model you have created in your module this problem exists
Please fix this